ksql-parser/src/main/java/io/confluent/ksql/parser/DefaultKsqlParser.java

```diff
@@ -74,16 +74,16 @@ public PreparedStatement<?> prepare(
       final AstBuilder astBuilder = new AstBuilder(metaStore);
       final Statement root = astBuilder.build(stmt.getStatement());
 
-      return PreparedStatement.of(stmt.getStatementText(), root);
+      return PreparedStatement.of(stmt.getUnMaskedStatementText(), root);
     } catch (final ParseFailedException e) {
       if (!e.getSqlStatement().isEmpty()) {
         throw e;
       }
       throw new ParseFailedException(
```

Comment:
Same comment as `KsqlStatementException`. Maybe safer to pass `stmt` and call `getMaskedStatementText` inside. Maybe not in this PR if it's too much change.